debian/arduino-app-cli/usr/sbin/arduino-avahi-serial.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# Configure Avahi with the serial number.
+# This operation is non-blocking: if it fails,
+# the script will exit with success in order to 
+# not to interrupt the post-install process.
```

Comment:
I think this script should and could fail. If there is an issue, I would prefer that it start the next time instead of never